carbon-design-system/carbon
# fix(ui-shell): a11y updates, refactor right panel

Semi-related to https://github.com/carbon-design-system/carbon/issues/3195

Ensured components can receive `aria-label` and `aria-labelledby`

**Added** 
- Keyboard navigation for right panel items
- `role` and `aria-labels` to some components, as well as in the stories to resolve console errors
- `menuLinkName="Link 4"` to a few `HeaderMenu` that were missing the prop

**Changed** 
- `SwitcherItem` now imports `Link` and handles everything `SwitcherItemLink` did, mimicking `HeaderMenuItem` 
- Changed a few `propTypes` to handle the refactor 

**Removed**
- Removed the `SwitcherItemLink`  component, and moved it inside of `SwitcherItem`. `SwitcherItem` now behaves just like `HeaderMenuItem`, so it should cause less friction. 



### **Testing**
- Ensure the Right Panel menu items can be tabbed to
- Ensure aria-labels are able to be passed through
